docs/concepts/controller-manager.md

```diff
@@ -84,6 +85,12 @@ The component configuration of the Gardener Controller Manager offers to configu
 
 > Gardener administrators/operators can exclude specific `Project`s from the stale check by annotating the related `Namespace` resource with `project.gardener.cloud/skip-stale-check=true`.
 
+#### "Activity" Reconciler
+
+Since the other two reconcilers are unable to actively monitor the relevant objects that are used in a `Project` (`Shoot`, `Plant`, etc.), there could be a situation where the user creates and deletes objects in a short period of time. In that case the `Stale Project Reconciler` could not see that there was any activity on that project and it will still mark it as a `Stale`, even though it is actively used.
+
+The `Project Activity Reconciler` is implemented to take care of such cases. An event handler will notify the reconciler for any acitivity (Currently only for `Shoots`) and then it will update the `Project.Status.LastActivityTimestamp`. This update will also trigger the `Stale Project Reconciler`.
```

Comment:
```suggestion
The `Project Activity Reconciler` is implemented to take care of such cases. An event handler will notify the reconciler for any activity (currently only for `Shoots`) and then it will update the `Project.Status.LastActivityTimestamp`. This update will also trigger the `Stale Project Reconciler`.
```

Reply:
```suggestion
The `Project Activity Reconciler` is implemented to take care of such cases. An event handler will notify the reconciler for any acitivity (Currently only for `Shoots`) and then it will update the `.status.lastActivityTimestamp`. This update will also trigger the `Stale Project Reconciler`.
```